bot/core/events/registry.py

```
@@ -0,0 +1,468 @@
+"""
+Base event classes for the bot.
+
+This module provides the base Event class and related helpers for the event system.
+Events are used for communication between different components of the application without
+direct dependencies.
+"""
+
+from bot.core.events.base import Event, EventType
+from typing import Any, Dict, List, Optional, Set, Type, TypeVar, cast
```

Ruff (0.8.2)

`typing.Set` imported but unused

Remove unused import

(F401)